### DISCUSSION TITLE
How to change the triangle icon that alongside the node?

### DISCUSSION THREAD

**Discussion by reregaga** [freeplane/freeplane#797] [Help] [answered]
When I use the mindmap from Help -> Tutorial, it has a triangle icon to indicate that the node has a note:
>![freeplane](https://user-images.githubusercontent.com/76629445/199740974-c3c5ff15-1594-4e84-9132-f08ad4eff70d.png)

How can I change this triangle icon to a different icon? I can't find setting for that.

In my created map this icon looks different:
>![my](https://user-images.githubusercontent.com/76629445/199741151-5924d6a0-3da5-421f-9e5c-2bdc49380416.png)

What does the appearance of the icon depend on?

**Comment by euu2021** [comment, marked as answer]
The triangle indicates that the node is minimized (View->Minimize node).

**Reply by reregaga** [reply]
Thanks, now I see that I can turn off the triangle by right-mouse-clicking on a node and selecting `Minimize node (Alt+F3)`.
